src/Service/ProgrammesAggregationService.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace BBC\ProgrammesPagesService\Service;
+
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\ClipRepository;
+use BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository;
+use BBC\ProgrammesPagesService\Mapper\ProgrammesDbToDomain\CoreEntityMapper;
+use BBC\ProgrammesPagesService\Cache\CacheInterface;
+
+class ProgrammesAggregationService extends AbstractService
```

Comment:
Discussing with some people about it we thought that if we keep adding all the stuff in CoreEntitiesService this will grow up too much. But happy to move it back to CoreEntities if necessary.